targets/TARGET_STM/rtc_api.c

```diff
@@ -270,7 +270,7 @@ void rtc_write(time_t t)
 
 #if DEVICE_LPTICKER && !MBED_CONF_TARGET_LPTICKER_LPTIM
     /* Need to update LP_continuous_time value before new RTC time */
-    uint32_t current_lp_time = rtc_read_lp();
+    rtc_read_lp();
```

Comment:
Might be nice to (void) the return of this function to show that you are purposefully ignoring the return value ?

Reply:
In this very specific case, we don't need to get the return value.
Function is called because global variable LP_continuous_time is updated there.